app/service/auth-service.js

```diff
@@ -0,0 +1,80 @@
+/* global __API_URL__ */
+'use strict';
+
+const angular = require('angular');
+const brewBuddy = angular.module('brewBuddy');
+
```

Comment:
Greate use of promises, and error handling through out this singleton
also good use of consistant naming patterns